lib/cli/src/commands/run.rs

```diff
@@ -859,9 +859,22 @@ pub(crate) fn try_run_package_or_file(
     let package = format!("{}", r.path.display());
 
     let mut is_fake_sv = false;
-    let mut sv = match SplitVersion::parse(&package) {
```

Comment:
I would approach this differently: 

* first, try to normalize the argument with `Path::canonicalize()` 
* Then, check if it's a valid file (.wasm or .wat), and run it if that's the case
   (there is already similar logic above, but only for cases where there is a `wapm.toml`)

* If that doesn't work, try to parse it as a package identifier

Reply:
To add on top. If is not "parseable" as a package identifier, it should not fall back into the registry.

Basically, here's the logic:
* Does the file exists? If so, try to run it
* If the file doesn't exist. Does it look like a package name? If so, transform it to a url
* Is the provided path an url? Try to download it
* Is the provided path not an url, show an error mentioning the file path doesn't work

Reply:
We need to follow this logic

Reply:
Note for us in the future: we can enforce that namespaces exists always (for now), as a tradeoff towards simplification

Reply:
If no namespace is given, we assume the namespace as being `_`